internal/output/diff.go

```diff
@@ -229,6 +269,17 @@ func costComponentToDiff(currency string, diffComponent CostComponent, oldCompon
 		)
 	}
 
+	if showEmissions {
+		if oldEmissions == nil && newEmissions == nil {
+			s += "  Monthly emissions depends on usage\n"
```

Comment:
```suggestion
			s += "  Monthly emissions depend on usage\n"
```
[Non blocking] Same as above, it admittedly depends on how the sentence is heard.